substrate/frame/transaction-storage/src/lib.rs

```diff
@@ -168,9 +168,28 @@ pub mod pallet {
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
 		/// Maximum number of indexed transactions in the block.
+		#[pallet::constant]
 		type MaxBlockTransactions: Get<u32>;
 		/// Maximum data set in a single transaction in bytes.
+		#[pallet::constant]
 		type MaxTransactionSize: Get<u32>;
+		/// Authorizations expire after this many blocks.
+		#[pallet::constant]
+		type AuthorizationPeriod: Get<BlockNumberFor<Self>>;
+		/// The origin that can authorize data storage.
+		type Authorizer: EnsureOrigin<Self::RuntimeOrigin>;
+		/// Priority of store/renew transactions.
+		#[pallet::constant]
+		type StoreRenewPriority: Get<TransactionPriority>;
+		/// Longevity of store/renew transactions.
+		#[pallet::constant]
+		type StoreRenewLongevity: Get<TransactionLongevity>;
+		/// Priority of unsigned transactions to remove expired authorizations.
+		#[pallet::constant]
+		type RemoveExpiredAuthorizationPriority: Get<TransactionPriority>;
+		/// Longevity of unsigned transactions to remove expired authorizations.
+		#[pallet::constant]
+		type RemoveExpiredAuthorizationLongevity: Get<TransactionLongevity>;
```

Comment:
I get that you are copying this from a different repo, but if you not use them it is hard to judge. Generally I don't see the reason why this is exposed and should be configurable?